src/main/java/com/zaxxer/nuprocess/windows/WindowsProcess.java

```diff
@@ -712,7 +718,7 @@ private String getEnvironmentBlock(Map<String, String> env)
 
    private void checkHandleValidity(HANDLE handle)
    {
-      if (NuWinNT.INVALID_HANDLE_VALUE.getPointer().equals(handle)) {
+      if (NuWinNT.INVALID_HANDLE_VALUE.getPointer().equals(handle.getPointer())) {
```

Comment:
This was an unrelated issue I noticed while in the code. I don't see how the previous code could ever have correctly detected an invalid handle